app/src/main/java/org/wikipedia/gallery/GalleryActivity.kt

```diff
@@ -361,13 +361,6 @@ class GalleryActivity : BaseActivity(), LinkPreviewDialog.LoadPageCallback, Gall
         outState.putInt(KEY_PAGER_INDEX, binding.pager.currentItem)
     }
 
-    override fun onBackPressed() {
-        if (TRANSITION_INFO != null) {
-            showTransitionReceiver()
-        }
```

Comment:
The transition animation is missing from this change. Would it be possible to bring this back with this change?